### ISSUE TITLE
angular-meteor-auth doesn't pass minification

### ISSUE THREAD

**Issue by ThomsC** [Urigo/angular-meteor#1030]
Hi,

I use the package `angular-meteor-auth`. It works perfectly in local but when I deploy my app crashes because of an undefined provider. When I remove the package everything works great.

So I put the directive `ng-strict-di` to see what is going on and here is the error better explain.

```javascript

Error: [$injector:strictdi] function($auth, $rootScope) is not using explicit annotation and cannot be invoked in strict mode
http://errors.angularjs.org/1.4.8/$injector/strictdi?p0=function(%24auth%2C%20%24rootScope)
Error: [$injector:strictdi] function($auth, $rootScope) is not using explicit annotation and cannot be invoked in strict mode
http://errors.angularjs.org/1.4.8/$injector/strictdi?p0=function(%24auth%2C%20%24rootScope)
    at http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:91:12
    at Function.annotate [as $$annotate] (http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:3815:17)
    at Object.invoke (http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:4524:36)
    at http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:4367:79
    at forEach (http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:363:20)
    at createInjector (http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:4367:3)
    at doBootstrap (http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:1699:20)
    at bootstrap (http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:1720:12)
    at angularInit (http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:1614:5)
    at HTMLDocument.<anonymous> (http://localhost:3000/packages/angular_angular.js?6d4ee2c049f945acd899d01d101da60606fc5cc8:29036:5)

```
https://github.com/Urigo/angular-meteor/commit/7c26c5ed04061a8f30a11f26fcfe1de4b48af853#diff-12f582bdd45cb65d0867652fee518197R155

I think It should be ['$auth', '$rootScope', function ($auth, $rootScope) { ... } ] then it can be minify correctly.

https://github.com/Urigo/angular-meteor/commit/7c26c5ed04061a8f30a11f26fcfe1de4b48af853#diff-12f582bdd45cb65d0867652fee518197R64

And here too ['$q', '$rootScope', '$timeout', function ($q, $rootScope, $timeout) { ... } ]

**Comment by ThomsC** [comment]
as the pull request has been accepted I close

**Comment by cubitworx** [comment]
@ThomsC Could you please reference the pull request you are referring to in your last comment. Thanks!

**Comment by cubitworx** [comment]
Hopefully the following will help someone else...

I was adding the `angular` package to my meteor application expecting `ng-annotate` to be automatically included so that I did not have to provide annotations; as per [the tutorial](http://www.angular-meteor.com/tutorials/socially/angular1/dynamic-template).

However, I was still coming up against the same `explicit annotation` error as listed above. Turns out the version resolution was including angular@1.0.9 which does not automatically include `ng-annotate`.

Not sure at what version it was introduced but once I explicitly set the version to angular@1.2.1 it worked as expected.
